_docs/development/environment.md

```diff
@@ -74,50 +93,49 @@ First, install nodenv.
 brew install nodenv
 ```
 
-Second, adjust your [.bash_profile file](https://www.gnu.org/software/bash/manual/html_node/Bash-Startup-Files.html) so that nodenv is initialized when you open a new terminal session. If you don't use Bash, consult [nodenv's documentation](https://github.com/nodenv/nodenv#basic-github-checkout).
+Second, adjust your profile so that nodenv is initialized when you open a new terminal session. Follow the directions given from running:
 
 ```
-touch ~/.bash_profile
-echo 'eval "$(nodenv init -)"' >> ~/.bash_profile
-source ~/.bash_profile
+nodenv init
 ```
 
 Third, install Node.
 
 ```
-nodenv install 8.11.2
+nodenv install 12.14.0
 ```
 
 Fourth, install Yarn.
 
 ```
-brew install yarn --without-node
+source $profile
+brew install yarn
 ```
 
 
 ### Install Postgres
 
 Manifold stores most data in a PotsgreSQL database.
 
-As of November 2018, our Manifold packages include Postgres 9.6.9. This version changes periodically. The current minimum version is recorded in our [Omnibus packaging repository](https://github.com/ManifoldScholar/omnibus-manifold/blob/master/config/software/postgresql.rb#L18).
+As of January 2020, our Manifold packages include Postgres 12.1. This version changes periodically. The current minimum version is recorded in our [Omnibus packaging repository](https://github.com/ManifoldScholar/omnibus-manifold/blob/master/config/software/postgresql.rb#L18).
 
 First, install PostgreSQL.
 
 ```
 brew install postgres
 ```
 
-Second, use [Homebrew Services](https://github.com/Homebrew/homebrew-services) to run PostgreSQL in the background.s
+Second, use [Homebrew Services](https://github.com/Homebrew/homebrew-services) to run PostgreSQL in the background.
 
 ```
 brew services start postgresql
 ```
 
 ### Install Redis
 
-[Redis](https://redis.io/) is an open source (BSD licensed), in-memory data structure store, used primarily by Manifold as a cache and message broker. Manifold is compatible with Redis v3 and v4. It is likely compatibile with v5, as well, although we have not formally tested it against v5.
+[Redis](https://redis.io/) is an open source (BSD licensed), in-memory data structure store, used primarily by Manifold as a cache and message broker. Manifold is compatible with Redis v3, v4 and v5.
```

Comment:
`brew install redis` will install version 5.0.7.

Should I say that manifold is compatible with v3 through v5?